### ISSUE TITLE
F1 Championship Season 2000 - Won't boot past Loading screen. 

### ISSUE THREAD

**Issue by MLH199** [pathartl/BleemSync#211]
Definately the best F1 game on the PS1 and not to be confused with F1 2000. But the issue is that the game loads, plays the video and then can't get past the FIA loading screen. Tried lot's of different roms, all give the same outcome. Can anyone help with this. The same Rom images work on my PC using ePSXe205. All other games, GT2, Porche Challenge, you name it work. 

Thx

**Comment by MLH199** [comment]
![f1_championship_season_2000_frontcover_large_xou5ija34lm21vd](https://user-images.githubusercontent.com/46301187/51031616-0cc41c00-1595-11e9-9cc2-3e4f439c8e4b.jpg)

**Comment by andshrew** [comment]
Best F1 game? That's a funny way of typing Formula 1 '97.

I would suggest trying the game on your PC using the RetroArch PCSX ReARMed core, this will give you a better indication as to whether this is a game the PSC emulator can support compared with trying it on a completely unrelated emulator ePSXe.

**Comment by MLH199** [comment]
> Best F1 game? That's a funny way of typing Formula 1 '97.
> 
> I would suggest trying the game on your PC using the RetroArch PCSX ReARMed core, this will give you a better indication as to whether this is a game the PSC emulator can support compared with trying it on a completely unrelated emulator ePSXe.

Yes, F1 97 is great, a great arcade game. But the F1 game I mentioned is like the F1 of Gran Turismo. Unfortunately, a lot of people skipped it and moved on by this point. Opinions eh?  

The game works fine on the emulators mentioned, it must be an issue with the emulation on the Playstation Classic. Maybe a bios file issue or something.

**Comment by andshrew** [comment]
Interesting you can get it working on RetroArch PCSX ReARMed core, as I've tried the NTSC version on both a PC and the PSC and in both cases the game will hang on the Licensed by Formula One Administration Limited screen - which would suggest is is an incompatibility with the emulator.

**Comment by MLH199** [comment]
Yep, same for me. I'm new to PS1 emulation, so used something called Beetle thinking it was PCSX. Buggered then! Would have loved to have played it on my Mini. I wonder if the guys who write and maintain the emulator could help?

**Comment by andshrew** [comment]
Best shot at it getting fixed would be raising it as an issue over at the PCSX ReARMed repository, and if it is fixed at the source then you would be able to play it on the PS Classic via RetroArch.

https://github.com/libretro/pcsx_rearmed/issues

**Comment by MLH199** [comment]
I will do that and thanks for your help, much appreciated.

**Comment by fawer5dev** [comment]
Hello @MLH199 Could you fix it? I also want to play this f1, please, let me know to mail   fawer5@   hotmail.com
Thanks.
